presto-orc/src/test/java/com/facebook/presto/orc/metadata/statistics/TestIntegerStatisticsBuilder.java

```diff
@@ -147,6 +153,33 @@ public void testMergeOverflow()
         assertMergedIntegerStatistics(statisticsList, 2, null);
     }
 
+    @DataProvider
+    public static Object[][] addValueByPositionDataProvider()
+    {
+        return new Object[][] {
+                {SmallintType.SMALLINT, SmallintType.SMALLINT.createBlockBuilder(null, 3).writeShort(3).appendNull().writeShort(10).build()},
+                {IntegerType.INTEGER, IntegerType.INTEGER.createBlockBuilder(null, 3).writeInt(3).appendNull().writeInt(10).build()},
```

Comment:
I could not do static import from the IntegerType.INTEGER because there is already static import for StatisticsType.INTEGER